draftquality/feature_lists/enwiki.py

```diff
@@ -1,6 +1,40 @@
 from revscoring.features import wikitext
 from revscoring.features.modifiers import max, sub
 from revscoring.languages import english
+from revscoring import Feature
+from revscoring.datasources import Datasource
+from senti_classifier import senti_classifier
+
+def get_polarity_score(revision_text):
+    """
+    Gets the positive and negative polarity of the document using SentiWordnet
+    limits the document to 20 sentences for efficiency
+    """
+    doc = revision_text.lower().split("\n")
+    return senti_classifier.polarity_scores(doc[:20])
```

Comment:
I'll see if I can make any quick performance improvements and ping again, before you test on a larger dataset.